Decentralization - Using Pledge as a Bidding Parameter to Determine Saturation Limit.md

```diff
@@ -0,0 +1,233 @@
+CIP: TBD\
+Title: Decentralization: Using Pledge as a Bidding Parameter to Determine Saturation Limit\
+Author: Jay Pseudonym Cappucino \<jycappucino@gmail.com>\
+Status: Draft\

```

Comment:
Since your proposal has already had some prelimintary discussion on the Cardano Forum (https://forum.cardano.org/t/cip-stake-decentralization-using-pledge-as-a-bidding-parameter-to-determine-saturation-limit/95936), would you include this link as a `Comments-URI:` header item? (to promote user input & discussion)